proto/p4/v1/p4runtime.proto

```diff
@@ -210,6 +210,18 @@ message TableEntry {
   IdleTimeout time_since_last_hit = 10;
   // Arbitrary metadata from the controller that is opaque to the target.
   bytes metadata = 11;
+  // const is true only for entries that cannot be deleted or
+  // modified, e.g. any of the following:
+  // + Any entry read from a table declared with `const entries`
+  // + The default entry read from a table declared with `const default_action`
+  // + Any entry declared with `entries` without the `const` qualifier
+  //   before `entries`, and thus the table has `has_initial_entries =
+  //   true`, where the individual entry has the `const` qualifier on
+  //   it in the P4 source code.
+  // Note: Older P4Runtime API servers before the `const` field was
+  // added to this message will not return a value for `const` in the
+  // first two cases above, but newer P4Runtime API servers will.
+  bool const = 13;
```

Comment:
Nit: Any thoughts on `const` vs `is_const`? Googlers would probably generally prefer the latter, and I also have a slight preference that way.

Reply:
I don't have a strong preference either way, so fine with changing it to `is_const`

Reply:
Note: I have changed it to `is_const` last Friday, so resolving this comment.